config/tcs_config.toml

```diff
@@ -66,8 +66,8 @@ WorkOrderSyncUri = "http://localhost:8080"
 WorkOrderAsyncUri = "http://localhost:8080"
 WorkOrderPullUri = "http://localhost:8080"
 WorkOrderNotifyUri = "http://localhost:8080"
-ApplicationTypeId = "Application_01"
```

Comment:
Instead adding hard coded random value, can you add something like in the python code.
"Application_01".encode("UTF-8").hex()
"Organization_01".encode("UTF-8").hex()

Reply:
If enclave manager does the hex conversion, aren't we modifying client specific config?
As per spec, users need to input their applicationtypeId and OrganizationId in hex format. I will add this comment.